<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiltracionDatos_v001</class>
 <widget class="QMainWindow" name="FiltracionDatos_v001">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FiltracionDatos_v001</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../Downloads/filtrar.png</normaloff>../../../../../Downloads/filtrar.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow {
  background-color: #2b2b2b;
  color: #f5f5f5;
}

QLabel {
  color: #f5f5f5;
}

QPushButton {
  background-color: #007aff;
  color: #f5f5f5;
  border: none;
  padding: 8px 16px;
  border-radius: 4px;
}

QPushButton:hover {
  background-color: #0059a6;
}

QPushButton:pressed {
  background-color: #003d79;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>67</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="topMargin">
      <number>7</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Cedula</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>DIreccion</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Telefono</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Ciudad</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Correo E.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>261</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  background-color: #FFFFFF;
  color: #000000;
  border: none;
  padding: 8px 16px;
  border-radius: 4px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>320</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Filtrar informacion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Abrir Archivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>22</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditNombre"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCedula"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDireccion"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTelefono"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCiudad"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCorreo"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_del_programa"/>
   </widget>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAcerca_del_programa">
   <property name="text">
    <string>Acerca del programa...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
